src/btree/bt_split.c

```diff
@@ -1414,6 +1414,9 @@ __split_multi_inmem(WT_SESSION_IMPL *session, WT_PAGE *orig, WT_MULTI *multi, WT
     if (!__wt_readgen_evict_soon(&orig_read_gen))
         __wt_atomic_store64(&page->read_gen, orig_read_gen);
 
+    if (F_ISSET(S2BT(session), WT_BTREE_GARBAGE_COLLECT))
```

Comment:
The title of the PR says it should only do this if the page is not reconciled to an empty page. Where is that check done? It doesn't seem to be in `__split_multi_inmem`, is there another part of the reconciliation code where this happens?

Reply:
Here we are reading the reconciled image to memory. We are only here when the page is not empty.

Reply:
Got it - thanks.